### DISCUSSION TITLE
ダークモードを選択できるようにしてほしい

### DISCUSSION THREAD

**Discussion by shunsuke250** [increments/qiita-discussions#546] [要望・アイデア・不具合]
現在ダークモードにするにはシステムのテーマをダークに設定する必要があります。
システム設定は自動の状態で、Qiitaの表示のみダークモードに設定したいです。

**Comment by tanaka-asami415** [comment]
@shunsuke250 
ご意見ありがとうございます！
同様のご意見が多いようであれば検討したいと思います🙇
